cmake/TargetCapnpSources.cmake

```diff
@@ -61,15 +61,15 @@ function(target_capnp_sources target include_prefix)
   )
 
   set(MPGEN_BINARY "")
-  if(EXTERNAL_MPGEN)
-    set(MPGEN_BINARY "${EXTERNAL_MPGEN}")
+  if(Libmultiprocess_EXTERNAL_MPGEN)
+    set(MPGEN_BINARY "${Libmultiprocess_EXTERNAL_MPGEN}")
     if(NOT EXISTS "${MPGEN_BINARY}")
-      message(FATAL_ERROR "EXTERNAL_MPGEN: \"${MPGEN_BINARY}\" does not exist.")
+      message(FATAL_ERROR "Libmultiprocess_EXTERNAL_MPGEN: \"${MPGEN_BINARY}\" does not exist.")
     endif()
   elseif(TARGET Libmultiprocess::multiprocess)
-    set(MPGEN_BINARY $<TARGET_FILE:Libmultiprocess::mpgen>)
```

Comment:
I attempted this a bunch of different ways, and with some approach this was necessary, but I forget why. I assume this is fine, thanks for simplifying.